src/Bicep.Core/Syntax/SyntaxHelper.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Bicep.Core.Syntax
+{
+    public static class SyntaxHelper
+    {
+        public static IEnumerable<SyntaxBase>? TryGetAllowedItems(ParameterDeclarationSyntax parameterDeclarationSyntax)
```

Comment:
>TryGetAllowedItems [](start = 47, length = 18)

Should it be an extension method? Seems only used in 2 places, so maybe not?

Reply:
Only one in fact, so I'll leave as-is.